README.md

```diff
@@ -1,134 +1,145 @@
 # The API Console
 
-MuleSoft's API Console is a fully-fledged API documentation tool that generates mobile-friendly web-documentation based on RAML (Restful API Modeling Language) documents. It also provides a mechanism to let consumer directly interact with an API.
+MuleSoft's API Console is a full-fledged API documentation tool that generates mobile-friendly web documentation based on RAML (Restful API Modeling Language) documents. A mechanism provides direct interaction between the consumer and an API.
```

Comment:
Hi Kris, `A mechanism provides ...` doesn't seem to be right here. It should be a follow-up sentence that additionally to just providing documentation, it also gives users to ability to try it out requests on the fly.